defenseunicorns/uds-core
# feat: grafana-ha

## Description

- Document the HA setup requirements for Grafana. 
- Ensure that these are provided as an example configuration.
- Setup the nightly testing with an external RDS database to ensure this is consistently tested in nightly CI.
- Update task for testing HA configuration locally to utilize single postgres docker container for keycloak and grafana.

## To Test
1. utilize the `test-uds-core-ha` task to deploy HA grafana and keycloak.
2. after deployment, access grafana dashboard at grafana.admin.uds.dev and validate things are working.
3. make sure there is scaled grafana pods ( should be 2 pods ).
4. add a dashboard to grafana ui.
5. delete pods and validate dashboard still exists.

## Related Issue

Fixes #716

## Type of change


## Checklist before merging
